pkgs/os-specific/darwin/apple-sdk/sdkRoot.nix

```diff
@@ -41,28 +41,24 @@ runCommand "sdkroot-${sdkVersion}" { } ''
 
   install -D '${../../../build-support/setup-hooks/role.bash}' "$out/nix-support/setup-hook"
   cat >> "$out/nix-support/setup-hook" <<-hook
-  #
```

Comment:
So we're dropping the exception when the hook isn't a build-time dependency and we're not doing a native compile?

Seems like a good thing to have it apply consistently.

Reply:
It needs to be `depsHostTarget` (i.e., a `buildInput`), but the hook wasn’t being run unless I took this out. The doc seems to suggest it should work, but maybe I was doing something wrong.